Composer/packages/tools/language-servers/language-generation/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "@bfc/lg-languageserver",
+  "version": "0.1.0",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc --build tsconfig.build.json",
+    "build:demo": "cd demo && tsc --build tsconfig.json",
+    "prepublishOnly": "npm run build",
+    "clean": "rimraf lib demo/dist",
+    "start": "cd demo && ts-node ./src/server.ts",
+    "test": "jest",
+    "lint": "eslint --quiet --ext .js,.jsx,.ts,.tsx ./src ./__tests__",
+    "lint:fix": "yarn lint --fix",
+    "lint:typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@types/node": "^12.0.4",
+    "botbuilder-lg": "https://botbuilder.myget.org/F/botbuilder-declarative/npm/botbuilder-lg/-/4.7.0-preview2.tgz",
+    "request-light": "^0.2.2",
+    "vscode-languageserver": "^5.3.0-next"
```

```suggestion
    "vscode-languageserver": "5.3.0-next"
```

Use either `^5.3.0` or `5.3.0-next`. Don't use `^5.3.0-next` because it literally means any version.

Also, when shipping a product, consider using all production build of packages. Development build aren't as stable.